testing/src/main/java/org/oppia/android/testing/OppiaTestRule.kt

```diff
@@ -133,5 +234,34 @@ class OppiaTestRule : TestRule {
     private fun <T> Class<T>.areAccessibilityTestsEnabledForClass(): Boolean {
       return getAnnotation(DisableAccessibilityChecks::class.java) == null
     }
+
+    /**
+     * Extracts all feature flag annotations of the specified type from a collection of annotations.
+     *
+     * @param annotations a collection of annotations to be checked for feature flag annotations.
```

Comment:
Todo: To remove ending periods from all KDocs.
```suggestion
     * @param annotations a collection of annotations to be checked for feature flag annotations
```